src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -425,9 +426,23 @@ rct_window* WindowSceneryOpen()
     if (window != nullptr)
         return window;
 
+    // count scenery groups to determine window width
+    int windowWidth;
+    int numTabs = 0;
+    for (ObjectEntryIndex scenerySetIndex = 0; scenerySetIndex < MAX_TABS; scenerySetIndex++)
+    {
+        const auto* sceneryGroupEntry = get_scenery_group_entry(scenerySetIndex);
+        if (sceneryGroupEntry != nullptr && scenery_group_is_invented(scenerySetIndex))
+        {
+            numTabs++;
+        }
+    }
+    // calculate initial window width
+    windowWidth = std::max<int32_t>(WINDOW_SCENERY_MIN_WIDTH, 5 + numTabs * SCENERY_TAB_WIDTH);
```

Comment:
If possible, we always declare and assign in one statement. We also use `auto` for anything that can be inferred from the context.

```suggestion
    auto windowWidth = std::max<int32_t>(WINDOW_SCENERY_MIN_WIDTH, 5 + numTabs * SCENERY_TAB_WIDTH);
```